kore/test/Test/Kore/Equation/Application.hs

```diff
@@ -355,200 +354,6 @@ test_attemptEquation =
         (Pattern.fromTermLike a)
     ]
 
-test_attemptEquationUnification :: [TestTree]
```

Comment:
Why are all these tests being removed? Do they no longer apply?

Reply:
oh i guess these are to do with 
> removes tests which tested applying equations using unification

Reply:
They used to duplicate the other tests in that module which tested applying equations by matching.